src/journal/pcre2-dlopen.c

```diff
@@ -30,13 +30,13 @@ int dlopen_pcre2(void) {
         r = dlsym_many_and_warn(
                         dl,
                         LOG_ERR,
-                        &sym_pcre2_match_data_create, "pcre2_match_data_create_8",
-                        &sym_pcre2_match_data_free, "pcre2_match_data_free_8",
-                        &sym_pcre2_code_free, "pcre2_code_free_8",
-                        &sym_pcre2_compile, "pcre2_compile_8",
-                        &sym_pcre2_get_error_message, "pcre2_get_error_message_8",
-                        &sym_pcre2_match, "pcre2_match_8",
-                        &sym_pcre2_get_ovector_pointer, "pcre2_get_ovector_pointer_8",
+                        DLSYM_ARG(pcre2_match_data_create),
+                        DLSYM_ARG(pcre2_match_data_free),
+                        DLSYM_ARG(pcre2_code_free),
+                        DLSYM_ARG(pcre2_compile),
+                        DLSYM_ARG(pcre2_get_error_message),
+                        DLSYM_ARG(pcre2_match),
+                        DLSYM_ARG(pcre2_get_ovector_pointer),
```

Comment:
The suffix `_8` is dropped. Is it intended?

Reply:
Nope, this was not intended. Needs fixing. Will pus new version shortly.

Reply:
Oh, christ. THis actually *does* work correctly already. That's because pcre headers use macros to redirect xyz to xyz_8, and STRINGIFY as used in the DLSYM_ARG() honours that and resolves it. This means behaviour is actually already correct. I am kinda irritated by that though.

Will add a comment explaining this to the sources.